duo_client/admin.py

```diff
@@ -2838,6 +2840,8 @@ def update_integration(self,
         if name is not None:
             params['name'] = name
         if visual_style is not None:
+            warnings.warn("The 'visual_style' argument is deprecated. May be removed in a future release.",
```

Comment:
do we do this other places, for deprecated parameters?

Reply:
Yes. In get_authentication_log() and get_group()